content/academy/apify_platform/get_most_of_actors/seo_and_promotion.md

```diff
@@ -1,13 +1,13 @@
 ---
 title: SEO and promotion
 description: Optimize your actors to get more relevant visits from search engines like Google. Set search engine-friendly parameters and share your actor with the world.
-menuWeight: 5
+menuWeight: 3
 paths:
     # NOTE: IF ADDING A NEW PATH, LEAVE THE OLD ONES FOR REDIRECTS
     - actor/publishing/seo-for-actors
     - actors/publishing/seo-for-actors
     - actors/publishing/seo-and-promotion
```

Comment:
You can probably delete the old paths that were for docs - they don't work here anyway

Reply:
Done